File: week-01/mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int num;

    do
    {
        num = get_int("Height: ");
    }
    while (num <= 0);

    for (int i = 0; i < num; i++)
    {

        for (int j = 1; j < num - i; j++)
        {
            printf(" ");
        }

        for (int k = 0; k < i + 1; k++)
        {
            printf("#");
        }

        printf("\n");
    }
}

File: week-06/sentimental-mario-more/mario.py
from cs50 import get_int

height = get_int("Height: ")
while (height > 8 or height < 1):
    height = get_int("Height: ")

for h in range(height):
    espaco = height - 1 - h
    tijolo = height - espaco
    print(f'{" " * espaco}{"#" * tijolo}')
